package.json

```diff
@@ -40,6 +40,7 @@
     "babel-jest": "23.6.0",
     "babel-loader": "8.0.4",
     "compressjs": "1.0.3",
+    "cross-env": "5.2.0",
```

Comment:
Can you also run `yarn install` to get `yarn.lock` to update?

Reply:
done!

Reply:
Our CI should check for this...